MaxEnt.py

```diff
@@ -0,0 +1,72 @@
+from platform import node
```

Comment:
These first two lines should result in error (as we don't have these listed as dependencies).